programming/sr/cheat_sheet.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: page
+title: API Quick Reference
+---
+
+# SR API Quick Reference
+
+This page contains a quick guide to the `sr.robot3` API.
+
+For more information, make sure you check the rest of the documentation.
+
+## Initialising your robot
+
+### Standard Initialisation
+
+~~~~~ python
+R = Robot()
+~~~~~
+
+### Initialisation with manual start button
```

Comment:
I can't think of a better name, but "manual start button" didn't immediately feel clear to me what this was (from skimming the titles). Perhaps "Initialisation without waiting for the start button"?

Reply:
94317400c17acb078e8de9f4701471094c448f02